### DISCUSSION TITLE
Can't start the docmost project

### DISCUSSION THREAD

**Discussion by vc-bhanudatsinh** [docmost/docmost#264] [Help / Questions]
![image](https://github.com/user-attachments/assets/5075083c-ce58-4ab6-859f-5a1f8ce1c9c1)

I have install all the dependency of nx and nx cli. also tried nx reset but still same issue.

**Comment by GerardPolloRebozado** [comment]
try closing the code editor and using the file explorer to delete .nx/cache folder manually

**Comment by Philipinho** [comment]
@vc-bhanudatsinh do you still experience this issue?
